packages/config/config/devices/0x0063/14297_zw1002.json

```diff
@@ -7,10 +7,26 @@
 		{
 			"productType": "0x4952",
 			"productId": "0x3233"
+		},
+		{
+			"productType": "0x4952",
+			"productId": "0x3236"
 		}
```

Comment:
@JascoProducts can you confirm if these two are actually the same product from a Z-Wave perspective? Do both have the LED indicator config param?

Reply:
Are both 500 series?

Reply:
> Are both 500 series?

Yes.

Reply:
Thanks, then we can treat them as identical